app/views/candidate_interface/training_with_a_disability/edit.html.erb

```diff
@@ -14,16 +14,16 @@
         </legend>
 
         <p class="govuk-body">You might benefit from extra support if you’re disabled, have a mental health condition or educational needs.</p>
-        <p class="govuk-body">If you choose to tell us you need support, we’ll let your training provider know. They can then make adjustments so you can attend an interview or do the training.</p>
-        <p class="govuk-body">Examples of support could be:</p>
+        <p class="govuk-body">We can let your training provider know what support you need, so you can attend an interview or do the training.</p>
+        <p class="govuk-body">Examples of extra support could include:</p>
         <ul class="govuk-list govuk-list--bullet">
           <li>organising equipment like a hearing loop or an adapted keyboard</li>
           <li>putting you in touch with support staff if you have a mental health condition</li>
           <li>making sure classrooms are wheelchair accessible</li>
         </ul>
-        <p class="govuk-body">If the help you need is not covered by your provider making adjustments, you might also be able to get support from <%= govuk_link_to 'Access to Work', 'https://www.gov.uk/access-to-work' %>. This could include a grant to help cover the costs of practical support in the workplace.</p>
-        <h2 class="govuk-heading-m">It’s against the law to discriminate</h2>
-        <p class="govuk-body">If you’re disabled, <%= govuk_link_to 'it’s against the law to discriminate against you', 'https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability' %>. Training providers must not:</p>
+        <p class="govuk-body">You may also be able to get support from <%= govuk_link_to 'Access to Work', 'https://www.gov.uk/access-to-work' %> to help with workplace adjustments.</p>
+        <p class="govuk-heading-m">It's against the law for training providers to <%= govuk_link_to 'discriminate against you', 'https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability' %>.</p>
```

Comment:
```suggestion
        <p class="govuk-heading-m">It’s against the law for training providers to <%= govuk_link_to 'discriminate against you', 'https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability' %>.</p>
```

Reply:
Hi Paul,

I've now committed this suggested change for review.

Thanks

George